### PR TITLE
Refactor Operation Builder Tests to Compile GraphQL Operations

### PR DESCRIPTION
**Pull request by AnthonyMDev**

### CODE REVIEW

File: Sources/ApolloCodegenLib/Frontend/dist/ApolloCodegenFrontend.bundle.js [hunk 1]
```diff
@@ -1 +1 @@
...
...r.operationTypes)&&void 0!==n?n:[];for(const n of e)t[n.operation]=b(n.type)}return t}function b(e){var t;const n=e.name.value,r=null!==(t=Vn[n])&&void 0!==t?t:h[n];if(void 0===r)throw new Error(`Unknown type: "${n}".`);return r}function O(e){return e.kind===u.LIST_TYPE?new GraphQLList(O(e.type)):e.kind===u.NON_NULL_TYPE?new GraphQLNonNull(O(e.type)):b(e)}function S(e){const t=Object.create(null);for(const i of e){var n;const e=null!==(n=i.fields)&&void 0!==n?n:[];for(const n of e){var r;t[n.name.value]={type:O(n.type),description:null===(r=n.description)||void 0===r?void 0:r.value,args:L(n.arguments),deprecationReason:Mn(n),astNode:n}}}return t}function L(e){const t=null!=e?e:[],n=Object.create(null);for(const e of t){var r;const t=O(e.type);n[e.name.value]={type:t,description:null===(r=e.description)||void 0===r?void 0:r.value,defaultValue:rn(e.defaultValue,t),deprecationReason:Mn(e),astNode:e}}return n}function A(e){const t=Object.create(null);for(const i of e){var n;const e=null!==(n=i.fields)&&void 0!==n?n:[];for(const n of e){var r;const e=O(n.type);t[n.name.value]={type:e,description:null===(r=n.description)||void 0===r?void 0:r.value,defaultValue:rn(n.defaultValue,e),deprec
```
Comment (calvincestari) [inline Sources/ApolloCodegenLib/Frontend/dist/ApolloCodegenFrontend.bundle.js:1]: What changed in the JS bundle?

**Review by calvincestari — APPROVED** [review]
This is way more readable - nice work! Just the question on the bundle update otherwise I'm good with this PR.

File: Sources/ApolloCodegenLib/Frontend/dist/ApolloCodegenFrontend.bundle.js (hunk 1 above)
Comment (calvincestari) [reply Sources/ApolloCodegenLib/Frontend/dist/ApolloCodegenFrontend.bundle.js:1]: I'm an idiot! It was right there in the `index.js` file - `referencedTypes.add(typeCondition)`. 🤦🏻

Comment (AnthonyMDev) [reply Sources/ApolloCodegenLib/Frontend/dist/ApolloCodegenFrontend.bundle.js:1]: lol no worries! It’s just a bug fix. The commit messages for the commits that make the changes explain why.

**Review by designatednerd — COMMENTED** [review]
😻

**Pull request merged**